llpc/context/llpcCompiler.cpp

```diff
@@ -2837,12 +2837,20 @@ Result Compiler::buildRayTracingPipelineInternal(RayTracingContext &rtContext,
     StringRef fetchTrianglePosFunc = mainContext->getPipelineContext()->getRayTracingFunctionName(
         Vkgc::RT_ENTRY_FETCH_HIT_TRIANGLE_FROM_NODE_POINTER);
 
+    StringRef traceRayGetInstanceIndx =
```

Comment:
```suggestion
   auto fetchRayTracingFuncName = [&](StringRef attribute) -> StringRef { return mainContext->getPipelineContext()->getRaytracingFunctionName(attribute); };
   ...
    StringRef traceRayGetInstanceIndx = fetchRayTracingFuncName(Vkgc::RT_ENTRY_INSTANCE_INDEX);
    ...
```

To clean the code up a bit